cardano-crypto-class/src/Cardano/Crypto/Hash/Class.hs

```diff
@@ -183,7 +184,7 @@ hashFromBytesAsHex hexrep
 
 
 instance Show (Hash h a) where
-  show = show . hashToBytesAsHex
+  show = Text.unpack . Text.decodeUtf8 . hashToBytesAsHex
```

Comment:
Using `Data.Text.Encoding` to convert to string to avoid returning a the hash data as a string literal.